.gitignore

```diff
@@ -11,7 +11,7 @@ stats.json
 .DS_Store
 tsDist/
 dist/
-commons-dist/
```

Comment:
`commons-dist` is the build directory for `cbioportal-frontend-commons`, we should be still ignoring it

Reply:
fixed